sentinelhub/geopedia.py

```diff
@@ -388,15 +384,18 @@ def __len__(self):
     def _fetch_features(self):
         """ Retrieves a new page of features from Geopedia
         """
-        if self.next_page_url is None:
-            return
+        response = self.client.get_json(self.next, post_values=self.params, headers=self.gpd_session.session_headers)
```

Comment:
Are both `offset` and `limit` part of the `pagination['next']` link? Otherwise I think the `limit` gets lost...

Reply:
They do, I checked. But I agree it is a bit scary if this would ever accidentally get changed on Geopedia.

Reply:
Perhaps add a check and raise an error?

Reply:
I added a check that verifies these parameters exist in the next page URL.